### PR TITLE
Updated to utilize Conditional Operator.

### PR DESCRIPTION
**Pull request by AdamSpeight2008**
Utilize the conditional operator, simplify 3 line if - blocks into single line.

### CODE REVIEW

File: src/Compilers/CSharp/Portable/CSharpCompilationOptions.cs [hunk 1]
```diff
@@ -331,165 +257,87 @@ public new CSharpCompilationOptions WithPublicSign(bool publicSign)
...
         {
             comparer = comparer ?? AssemblyIdentityComparer.Default;
 
-            if (ReferenceEquals(comparer, this.AssemblyIdentityComparer))
-            {
-                return this;
-            }
-
-            return new CSharpCompilationOptions(this) { AssemblyIdentityComparer = comparer };
+            return (ReferenceEquals(comparer, this.AssemblyIdentityComparer)) ? this : new CSharpCompilationOptions(this) { AssemblyIdentityComparer = comparer };
```
Comment (jcouv) [inline src/Compilers/CSharp/Portable/CSharpCompilationOptions.cs]: Un-necessary parens, especially when no `==`.
Consider breaking long lines (I generally aim for 120-130 chars, but I don't think we have a hard rule).

File: src/Compilers/CSharp/Portable/CSharpCompilationOptions.cs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```
Comment (jcouv) [inline src/Compilers/CSharp/Portable/CSharpCompilationOptions.cs]: What changed here?
If unicode encoding changed, please revert. Apparently that causes pain during subsequent conflict resolutions.

Comment (AdamSpeight2008) [inline src/Compilers/CSharp/Portable/CSharpCompilationOptions.cs]: @jcouv This is 2nd time this has happened to me. It seems to happen when I use the built in edit feature on GitHub. (So maybe I'm missed setting some where.)

**Comment by jaredpar** [comment]
This is largely a style change to our code.  The style here is a bit verbose but it's pretty established and used in many places other than this.  Don't see a compelling reason to bring this in.  Closing for now.

**Pull request closed without merging**